modules/order/commerce_order.services.yml

```diff
@@ -9,3 +9,15 @@ services:
     arguments: ['@entity_type.manager']
     tags:
       - { name: commerce_order.order_type_resolver, priority: 10 }
+
+  commerce_order.order_refresh:
+    class: Drupal\commerce_order\OrderRefresh
+    arguments: ['@entity_type.manager']
+    tags:
+      - { name: service_collector, call: addProcessor, tag: commerce_order.order_refresh_processor }
```

Comment:
I copied how our other services.yml are, I'll update but we should update the others?